oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/AlarmCore.java

```diff
@@ -47,6 +47,14 @@
         return alarmRulesWatcher.getRunningContext().get(metricsName);
     }
 
+    /**
+     * Perform a series of tasks every 10 seconds, including update alarm settings dynamically,
+     * check alarm rules by check function, and calling doAlarm for alarm if trigger the
+     * rules (Send alarm message to webhook(s) through HTTP post).
```

Comment:
The ways of message forwarding depend on the implementation, we don't' restrict any thing here.